### ISSUE TITLE
Moving Obstacles within one map

### ISSUE THREAD

**Issue by abbas-tari** [ptsneves/rt-rrt#5]
Does this implementation cover moving obstacles within one map? If so, where exactly I can possibly see how moving obstacles are implemented?
-Thanks

**Comment by ptsneves** [comment]
No. The current implementation relies on a static map. You could think of obstacles as a new map at every trial, but i am not sure there are not better implementations and heuristics for such a case.
